docs/fundamentals/client/client.md

```diff
@@ -760,6 +760,34 @@ None
 
 ````
 
+A Client is used to connect to a Flow who processes Documents in an asynchronous and very distributed way. This means that the order in which the Requests are processed may not be the exact same order as they are sent by the Client.
```

Comment:
```suggestion
A Client connects to a Flow that processes Documents in an asynchronous and very distributed way. This means that the order of the Flow processing the requests may not be the same order as the Client sending the requests.
```
I think a short example would be useful here. Parsing that text was not simple.

Wait, when we say "who processes Documents in an asynchronous and very distributed way" does that refer to client or flow?

Maybe just rewrite this bit. There's ambiguity

Reply:
the Flow processes it in an asynchronous and distributed way

Reply:
There is a test already